platform/src/test/java/platformstepdefinition/PF_Footer.java

```diff
@@ -0,0 +1,51 @@
+package platformstepdefinition;
+
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+
+import base.NewBaseClass;
+import cucumber.api.PendingException;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.junit.Cucumber;
+import platform.pageobjects.Authentication.LoginPage;
+import platform.pageobjects.Footer.FooterPage;
+
+@RunWith(Cucumber.class)
+public class PF_Footer extends NewBaseClass {
+
+	public WebDriver driver;
```

Comment:
Replace with public WebDriver driver = ObjectHelper.driver;
Needed to retain browser instance

Reply:
Done changes as requested.